Presentation.Web/app/components/org/user/org-user-delete.controller.ts

```diff
@@ -69,7 +70,8 @@
             private readonly _: ILoDashWithMixins,
             text,
             allRoles: Models.Users.UserRoleAssigmentDTO,
-            private readonly userRoleAdministrationService: Services.IUserRoleAdministrationService) {
+            private readonly userRoleAdministrationService: Services.IUserRoleAdministrationService,
+            private readonly userService: Services.IUserService) {
```

Comment:
that service is mainly about the "logged in user".. so place it in another service (the method)

Reply:
There is actually one: `user-role-administration.service.ts`. It already calls the endpoint just without the org (for the global admin flow)